logica.py

```diff
@@ -194,17 +202,16 @@ def main(argv):
         o, _ = p.communicate(
             '\n'.join(commands + [formatted_sql]).encode())
       elif engine == 'trino':
-        a = p.annotations.annotations['@Engine']['trino']
-        catalog = a.get('catalog', 'memory')
```

Comment:
We should keep memory as a default value of catalog.

Reply:
I agree and given the other changes coming in by accident. I'll update the code taking taking into account the comments mentioned above.